src/System.Diagnostics.EventLog/tests/EventLogTests/EventLogSourceCreationTests.cs

```diff
@@ -1,12 +1,11 @@
-﻿using System.Diagnostics;
-using Xunit;
-using System;
+﻿using Xunit;
 
 namespace System.Diagnostics.Tests
 {
     public class EventLogSourceCreationTests : EventLogTestsBase
     {
-        [ConditionalFact(nameof(IsProcessElevated))]
+        //[ConditionalFact(nameof(IsProcessElevated))]
```

Comment:
Here you need to combine two conditions. You should be able to do
`[ConditionalFact(nameof(PlatformDetection.IsNotWindowsNanoServer), nameof(AdminHelpers.IsProcessElevated)]`